modules/tinymce/src/core/main/ts/keyboard/InsertSpace.ts

```diff
@@ -45,7 +49,24 @@ const insertSpaceOrNbspAtSelection = (editor: Editor): Optional<() => void> => {
   }
 };
 
+// TINY-9964: Firefox has a bug where the space key is toggling the open state instead of inserting a space in a summary element
+const insertSpaceInSummaryAtSelectionOnFirefox = (editor: Editor): Optional<() => void> => {
+  const insertSpaceThunk = () => {
+    const root = SugarElement.fromDom(editor.getBody());
+
+    if (!editor.selection.isCollapsed()) {
+      editor.getDoc().execCommand('Delete');
+    }
+
+    const pos = CaretPosition.fromRangeStart(editor.selection.getRng());
+    insertSpaceOrNbspAtPosition(root, pos).each((pos) => editor.selection.setRng(pos.toRange()));
```

Comment:
Wondering if we can use the `setSelection` function that `insertSpaceOrNbspAtSelection` uses
```suggestion
    insertSpaceOrNbspAtPosition(root, pos).each(setSelection);
```

Reply:
Right good point noticed that I returned true that is never used so I changed that as well.